pom.xml

```diff
@@ -34,6 +34,268 @@
 
     <name>Gravitee.io APIM</name>
 
+<<<<<<< HEAD
+=======
+    <properties>
+        <!-- Version properties -->
+        <revision>4.6.0</revision>
+        <sha1/>
+        <changelist>-SNAPSHOT</changelist>
+
+        <!-- Vert.X version is mandatory for vertx-grpc-protoc-plugin2
+             in gravitee-apim-gateway-tests-sdk and gravitee-apim-integration-tests
+             along with the grpc libs to create test clients and servers -->
+        <vertx.version>4.5.10</vertx.version>
+        <protobuf-java.version>4.28.2</protobuf-java.version>
+        <grpc-java.version>1.65.0</grpc-java.version>
+        <!-- Gravitee dependencies version -->
+        <gravitee-bom.version>8.2.0</gravitee-bom.version>
+        <gravitee-alert-api.version>2.0.0</gravitee-alert-api.version>
+        <gravitee-cockpit-api.version>3.7.0</gravitee-cockpit-api.version>
+        <gravitee-cloud-initializer.version>2.1.0</gravitee-cloud-initializer.version>
+        <gravitee-spec-gen-api.version>1.1.0</gravitee-spec-gen-api.version>
+        <gravitee-common.version>4.6.0</gravitee-common.version>
+        <gravitee-secret-api.version>1.0.0-alpha.3</gravitee-secret-api.version>
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Config Generic   
**Description:** Generic Config Credential   
**Severity:** HIGH   
**Match:**         <gravitee-secret-api.version>*************</gravitee-secret-api.version>  
[This comment was created by Aqua Pipeline]